GRAVEYARD.md

```diff
@@ -0,0 +1,20 @@
+# Socorro Code Graveyard
+
+This document records interesting code that we've deleted for the sake of discoverability for the future.
+
+## Various scripts
+
+* [Removal PR](https://github.com/mozilla-services/socorro/pull/6674)
+
+`bin/remove_field.py` which removes fields from raw crashes in crash storage and elasticsearch.
+
+`bin/permadelete_crash_data.py` which deletes crashes from crash storage and elasticsearch.
```

Comment:
I think we want to keep this one because we want to make sure it works over time in case we need it (though that's very infrequent).

Can you un-graveyard this one script, please?